gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-http-proxy/pom.xml

```diff
@@ -78,6 +78,13 @@
     </dependencies>
 
     <build>
+        <extensions>
+            <extension>
+                <groupId>kr.motd.maven</groupId>
+                <artifactId>os-maven-plugin</artifactId>
+                <version>1.6.2</version>
+            </extension>
+        </extensions>
```

Comment:
Is there any reason why we need this extension now?

Reply:
I assumed it was mandatory for get the ${os.detected.classifier}